### ISSUE TITLE
Cache is flushed immediately during cache/rebuild

### ISSUE THREAD

**Issue by asgeirn** [FINTprosjektet/fint-consumer-skeleton#3]
This flush means the cache is empty until the adapter has received and responded to the update event.

**Comment by fsjovatsen** [comment]
@asgeirn @jarlehansen Is this fixed?

**Comment by asgeirn** [comment]
No, this is still not solved.  The cause is in https://github.com/FINTlibs/fint-cache/issues/9
